test/features/paths_spec.rb

```diff
@@ -3,6 +3,13 @@
 class Paths < ActionDispatch::IntegrationTest
   def test_can_visit_the_root_path_without_authentication
     visit "/"
-    assert page_has_content? "Login with GitHub"
+    assert page.has_content? "Login with GitHub"
+  end
+
+  def test_can_visit_the_dashboard_if_authenticated
+    user = Person.create(first_name: 'Aaron', last_name: 'Wortham')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.